### PR TITLE
pybind11 v2.6.2, remove libiomp5md.lib

### LINKED ISSUE #2116: conda/win/2019.1/libiomp5md.lib is a binary file without source
I don't think `conda/win/2019.1/libiomp5md.lib` should belong in the git repo and/or the tarballs; it is a binary file without source code.

### PR DESCRIPTION
**Pull request by loriab**
## Description
Bump pb11 to v2.6.2, bump v2rdm to a working commit, closes #2116 

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Review by jturney — APPROVED** [review]

File: .azure-pipelines/azure-pipelines-windows.yml
Comment (JonathonMisiewicz) [inline .azure-pipelines/azure-pipelines-windows.yml:13]: What does the `conda.build` line do? Based on commit history, it looks like something with the OpenMP copy, but I want to make sure this is still necessary. Will approve once that's cleared up.

Comment (loriab) [reply .azure-pipelines/azure-pipelines-windows.yml:13]: the `conda.build` line is separate from the omp business. the boolean activates an extra step in the azure (all set up by Raimondas) to build the Windows conda package, spot test, and save the result as an artifact. I toggled it here to make sure the omp lib changes weren't scuppering the conda package, but you're right that it should be off again in general (adds ~25 min, iirc). But I'm putting off changing it until I see if there are merge conflicts to resolve at the same time.

**Review by JonathonMisiewicz — APPROVED** [review]

**Review by andysim — APPROVED** [review]
LGTM!

**Pull request merged**
